.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/psf/black
```

Comment:
Ruff is already a formatter, so no reason to use black as well.

Reply:
Oh, wow, lol. I looked back to when I last added ruff to our cookiecutter and the ruff changelog and it was prior to them adding formatting. I didn't even know, thanks for the heads up!